R/MsBackend.R

```diff
@@ -245,6 +253,10 @@ NULL
 #'
 #' - `length`: returns the number of spectra in the object.
 #'
+#' - `lengths`: gets the number of peaks (m/z-intensity values) per
+#'   spectrum.  Returns an `integer` vector (length equal to the
+#'   number of spectra). For empty spectra, `NA_integer_` is returned.
```

Comment:
Mh, does `lengths` really return `NA` for empty spectra? Not zero? Is `NA` a good choice here?

Reply:
Thanks for spotting this! The way it is implemented it would report `0` not `NA`. I'll fix the documentation.